client/js/components/statement/assessmentTable/DpExportModal.vue

```diff
@@ -103,24 +103,25 @@
 
             <fieldset
               v-if="options.pdf.exportTypes && exportChoice.pdf.template == 'condensed' && view == 'assessment_table'"
-              class="u-mb-0_5 u-pb-0_5">
+              class="mb-2 pb-2">
               <legend
                 class="sr-only"
                 v-text="Translator.trans('export.data')" />
               <dp-radio
-                class="mb-1"
                 id="pdfExportTypeStatementsOnly"
+                :checked="exportChoice.pdf.exportType === 'statementsOnly'"
```

Comment:
I notice that I'm not always happy with the attribute order we defined, e.g. for me, `checked` and `@change` belong together. I don't mean this as a change request, I'd just like to know your opinion on it.

Reply:
I see your point, but I personally am fine with the alphabetic order. Other props may also belong together with events or other things and I think it could get confusing how to group and order them.